lib/material/extra/calendar/calendar.flow

```diff
@@ -3664,7 +3741,11 @@ CalendarForEvents(
 				}));
 
 				headerTextM = MSelect(currentDate, \date -> {
-					defaultText = formatString("WEEK %1 OF %2", [i2s(getWeekOfYear(date, 0, true)), i2s(date.year)]);
+					defaultText = if (containsStruct(styles, ShowHoursGridForWeekView(0.0))) {
```

Comment:
Could you use CalendarHeaderText style in nnc to change text?

Reply:
Done. Anything else?